app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
```

Comment:
```suggestion
       self.alive.append(self)
```

Reply:
is there some principal difference in these two cases?